tests/queries/0_stateless/03545_settings_over_cgi_params.reference

```diff
@@ -0,0 +1,8 @@
+output_format_parallel_formatting=1
+output_format_parallel_formatting	1	0
+output_format_parallel_formatting=0
+output_format_parallel_formatting	0	1
+output_format_parallel_formatting=1&output_format_parallel_formatting=0
+output_format_parallel_formatting	0	1
+output_format_parallel_formatting=0&output_format_parallel_formatting=1
+output_format_parallel_formatting	0	1
```

Comment:
Here is a problem:
URI `/?output_format_parallel_formatting=0&output_format_parallel_formatting=1`
does not change the value.